src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlConnectionString.cs

```diff
@@ -301,6 +307,15 @@ internal SqlConnectionString(string connectionString) : base(connectionString, G
             string typeSystemVersionString = ConvertValueToString(KEY.Type_System_Version, null);
             string transactionBindingString = ConvertValueToString(KEY.TransactionBinding, null);
 
+            string clientCertificate = ConvertValueToString(KEY.ClientCertificate, null);
+            string clientKey = ConvertValueToString(KEY.ClientKey, null);
+            string clientKeyPassword = ConvertValueToString(KEY.ClientKeyPassword, null);
+            if(clientCertificate != null)
+            {
+                //_clientCertificate = X509Certificate2PrivateKeyExtensions.ParseWithPrivateKey(clientCertificate, clientKey, clientKeyPassword);
```

Comment:
Why is this commented?

Reply:
X509Certificate2PrivateKeyExtensions class is beeing worked on its not jet pushed